MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -591,5 +594,29 @@ - (void)locationManager:(MMELocationManager *)locationManager didVisit:(CLVisit
     }
 }
 
+- (NSString *)locationAuthorizationStatus {
+    CLAuthorizationStatus status = [CLLocationManager authorizationStatus];
+    switch (status) {
+        case kCLAuthorizationStatusDenied:
+            return MMEEventStatusDenied;
+            break;
```

Comment:
return then break seems awkward, I'd recommend using a local variable to hold the result string, and having a single return (this was SoP at Apple, not sure what the local standard is).

Reply:
Good catch, been a while since I've used switch!